src/Entity/OgMembership.php

```diff
@@ -255,7 +255,8 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
     $fields['uid'] = BaseFieldDefinition::create('entity_reference')
       ->setLabel(t('Member entity ID'))
       ->setDescription(t('The entity ID of the member.'))
-      ->setTargetEntityTypeId('user');
+      ->setSetting('target_type', 'user')
+      ->setDefaultValue(0);
```

Comment:
Good point, I stole this from comment module, probably not the best idea. I will just remove the default value. Looks like we don't have that check in here yet, should I add it now?